src/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs

```diff
@@ -104,15 +105,35 @@ public void OpenFile_ThrowsIOException()
             }
         }
 
-        [ActiveIssue(4605, PlatformID.OSX)]
```

Comment:
Have we verified this fixes it on OS X?  Since the PR CI isn't currently validating OS X, we'd need to be prepared for this to fail and need a follow-up PR to disable it again.